themes/atomic/atomic.theme.bash

```diff
@@ -183,7 +224,27 @@ ___atomic_prompt_battery() {
 	info+=$batp
 	[ "$info" == "+100" ] && info="AC"
 	printf "%s|%s|%s|%s" "${color}" "${info}" "${bold_white}" "${box}"
-}
+	} ;; 
+  linux*) ___atomic_prompt_battery() {
+	[ ! -e "$BASH_IT"/plugins/enabled/battery.plugin.bash ] ||
+	[ "${THEME_SHOW_BATTERY}" != "true" ] && return
+	batp=$(battery_percentage)
+	if [ "$batp" -gt 50 ]; then
```

Comment:
The majority of this code looks to be pretty much identical for all four of your cases. There's probably a smarter way to handle this without having the same code four times.